app/controllers/identifiers_controller.rb

```diff
@@ -23,7 +24,9 @@ def create
     if @identifier
       render plain: @identifier.identifier, status: :created, location: @identifier
     else
-      render build_error('Unable to mint a druid', :internal_server_error)
+      error_message = 'Unable to mint a druid'
```

Comment:
I know "Unable to mint a druid" is the existing message, but can we provide something more useful to HB?

Reply:
@justinlittman Not with the current implementation, I don't think so. I'm open to ideas! I can at least add a new issue for making this more robust, though, if that sounds acceptable.

Reply:
@justinlittman https://github.com/sul-dlss/suri-rails/issues/64